package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "knex-mysql-deadlock",
+  "version": "1.0.0",
+  "description": "A custom knex mysql dialect, which does deadlock retries",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Mindflash/knex-mysql-deadlock.git"
+  },
+  "author": "mindflash",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Mindflash/knex-mysql-deadlock/issues"
+  },
+  "homepage": "https://github.com/Mindflash/knex-mysql-deadlock#readme",
+  "dependencies": {
+    "knex": "^0.12.6"
```

Comment:
>it will have to have node-mysql installed no matter what

knex doesn't come with node-mysql; it's a peer dependency. This package would only work with node-mysql, so it seems only natural to list it as a peerDep.

If I somehow installed this while using knex with only postgres, npm would warn that this package is missing it's peer dependency on node-mysql. (Yes, I know it sounds silly; who's going to install this plugin with mysql in it's name... but it's still possible, especially it you talk about situations where local and production databases and therefore adapters may differ).